src/team/component/santa/spec-fetcher/src/fetcher.ts

```diff
@@ -101,6 +109,8 @@ const fetcher = async (opts: Options) => {
         );
         yield true;
         continue;
+      } else {
+        await fs.rmdir(`${opts.output}/${repository}`, { recursive: true });
```

Comment:
```suggestion
        await fs.rmdir(`${specOutputDir}`, { recursive: true });
```
97번 라인 변수 재사용하면 좋겠습니다!